### ISSUE TITLE
prerelease option is not working as expected

### ISSUE THREAD

**Issue by dkarkar92** [google-github-actions/release-please-action#653]
### TL;DR

Using the "prerelease" option you would expect that if your current latest release is `1.5.0`, and you add a new `feat: xyz` commit, the action should create a new `1.6.0-beta` release. This does not appear to work.

### Expected behavior

If you use the "prerelease=true" option, you would expect the next release to have a prerelease of `x.y.z-alpha` or something similar.

### Observed behavior

With the "prerelease=true" option, the version is always the same as if "prerelease=false". AKA `x.y.z` with no alpha / beta

### Action YAML

```yaml
on:
  push:
    branches:
      - test-dk-main
name: release-please-test
jobs:
  release-please-test:
    runs-on: self-hosted
    steps:
      - name: Checkout source
        uses: actions/checkout@v3

      - name: Read commit-types.json from configs and output for use in changelog-types
        id: changelog-types-output
        run: |
          echo "COMMIT_TYPES_STR=$(
            cat configs/commit-types.json| jq -r tostring
          )" >> $GITHUB_OUTPUT

      - uses: google-github-actions/release-please-action@v3.5.0
        with:
          release-type: node
          package-name: xyz-name
          token: ${{ secrets.RELEASE_TOKEN }}
          changelog-types: ${{ steps.changelog-types-output.outputs.COMMIT_TYPES_STR }}
          prerelease: true
          default-branch: test-dk-main
```

### Log output

```text
##[group]Run google-github-actions/release-please-action@v3.5.0
 with:
   release-type: node
   package-name: xyz-name
   token: ***
   changelog-types: [{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous Chores","hidden":false},{"type":"refactor","section":"Code Refactoring","hidden":false},{"type":"build","section":"Build System","hidden":false},{"type":"test","section":"Tests","hidden":false},{"type":"perf","section":"Performance Improvements","hidden":false},{"type":"revert","section":"Reverts","hidden":false},{"type":"docs","section":"Documentation","hidden":false},{"type":"style","section":"Styles","hidden":false},{"type":"ci","section":"Continuous Integration","hidden":false}]
   prerelease: true
   default-branch: test-dk-main
   fork: false
   clean: true
   bump-minor-pre-major: false
   bump-patch-for-minor-pre-major: false
   changelog-host: https://github.com
   github-api-url: https://api.github.com
   github-graphql-url: https://api.github.com/graphql
   monorepo-tags: false
 ##[endgroup]
 ✔ Looking for latest release on branch: test-dk-main with prefix: xyz-name
 ✔ Building releases
 ✔ Building strategies by path
 
 ##[warning]The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/-2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
 ✔ Looking for latest release on branch: test-dk-main with prefix: xyz-name
 ✔ Building pull requests
 ✔ Building strategies by path
 ✔ Collecting release commit SHAs
 ✔ Collecting commits since all latest releases
 ✔ Splitting 15 commits by path
 ✔ Building candidate release pull request for path: .
 ✔ Considering: 15 commits
 ✔ Looking for open release pull requests
 ✔ found 1 open release pull requests.
 ✔ Looking for snoozed release pull requests
 ✔ found 0 snoozed release pull requests.
 ⚠ file package-lock.json did not exist
 ⚠ file npm-shrinkwrap.json did not exist
 ⚠ file samples/package.json did not exist
 ✔ updating from 1.21.0 to 1.22.0
 ⚠ file .release-please-manifest.json did not exist
 ✔ Starting GitHub PR workflow...
 ✔ Successfully found branch HEAD sha "abcxxx".
 ✔ Skipping branch creation step...
 ✔ Got the latest commit tree
 ✔ Successfully created a tree with the desired changes with SHA abc123
 ✔ Successfully created commit. See commit at https://api.github.com/repos/xyz-name/git/commits/abc456
 ✔ Updating reference heads/release-please--branches--test-dk-main--components--xyz-name to abczzz
 ✔ Successfully updated reference release-please--branches--test-dk-main--components--xyz-name to abcyyy
 ✔ Found existing pull request for reference orgx:release-please--branches--test-dk-main--components--xyz-name. Skipping creating a new pull request.
 ✔ Successfully opened pull request: 897.
 
 ##[warning]The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/-2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
 
 ##[warning]The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/-2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

### Additional information

I tested via CLI from release-please itself and ran into the same issue. The "prerelease=true" does nothing via CLI either... which is dubious.

**Comment by chingor13** [comment]
The `prerelease` option does not affect the semver of the new client (there is no standard versioning flow for what the version should be).

From the `prerelease` docs: `If set, create releases that are pre-major or pre-release version marked as pre-release on Github. Defaults false`. This only affects whether the GitHub release is marked as "prerelease".

To control the version, you'd want to use a `Release-As: 1.2.3-beta` to inform release please of what the next version you want is.
